src/pages/RepoPage/ConfigTab/tabs/ConfigurationManager/components/FeatureGroup/FeatureGroup.tsx

```diff
@@ -1,10 +1,13 @@
+import { useNavLinks, useStaticNavLinks } from 'services/navigation'
 import A from 'ui/A'
 import Button from 'ui/Button'
 import { Card } from 'ui/Card'
 
 interface FeatureGroupProps extends React.PropsWithChildren {
   title: string
-  getStartedLink?: string // navLink key
+  getStartedLink?:
+    | keyof ReturnType<typeof useNavLinks>
```

Comment:
do we need that first `|`?

Reply:
Nope, it's optional - it's just for stylistic consistency.